doc/realworld.html

```diff
@@ -59,6 +59,7 @@
       <li><a href="https://www.codevolve.com/">Codevolve</a> (programming lessons as-a-service)</li>
       <li><a href="http://www.codezample.com">CodeZample</a> (code snippet sharing)</li>
       <li><a href="http://codio.com">Codio</a> (Web IDE)</li>
+      <li><a href="https://www.codiva.io/">Codiva.io</a> (Online Java Compiler and IDE with auto- completion and error highlighting)</li>
```

Comment:
I'm sorry, but there's still a small mistake. There's an extraneous space between ›auto‹ and ›completion‹, it should be ›auto-completion‹ without a space.
Also, can you squash your commits together into one commit so that our git history is tidy? Here's a [quick guide](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) how to do that.

Reply:
Thanks. I think I have fixed it correctly this time.